src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,10 @@
+import styles from './Auth.module.css'
```

Comment:
As mentioned. Can we make sure the SSI auth code is more clearly separated from the rest of the code. For instance by putting it all in a SSI subfolder like for instance src/components/ssi/auth etc

Reply:
Fixed